optuna/storages/_rdb/models.py

```diff
@@ -19,13 +19,19 @@
 from sqlalchemy import String
 from sqlalchemy import Text
 from sqlalchemy import UniqueConstraint
-from sqlalchemy.ext.declarative import declarative_base
 
 from optuna import distributions
 from optuna.study._study_direction import StudyDirection
 from optuna.trial import TrialState
 
 
+try:
+    from sqlalchemy.orm import declarative_base
+except ImportError:
+    # TODO(c-bata): Remove this after dropping support for SQLAlchemy v1.3 or prior.
+    from sqlalchemy.ext.declarative import declarative_base
```

Comment:
```
optuna/storages/_rdb/models.py:40
  /Users/c-bata/go/src/github.com/optuna/optuna/optuna/storages/_rdb/models.py:40: MovedIn20Warning: The ``declarative_base()`` function is now available as sqlalchemy.orm.declarative_base(). (deprecated since: 1.4) (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
    BaseModel: Any = declarative_base()
```